api/v1beta1/keystoneapi_types.go

```diff
@@ -197,6 +201,15 @@ type PasswordSelector struct {
 	Admin string `json:"admin"`
 }
 
+// HttpdCustomization - customize the httpd service
+type HttpdCustomization struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=3
+	// +kubebuilder:validation:Minimum=1
```

Comment:
as discussed on slack, this needs to be tested for updating the controller/CRD on an already deployed env